app/assets/stylesheets/application.scss

```diff
@@ -45,6 +45,9 @@ body {
   .container {
     padding-top: 20px;
   }
+  .input {
+    color: #fff;
```

Comment:
Color literals like `#fff` should only be used in variable declarations; they should be referred to via variable everywhere else.